src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -0,0 +1,165 @@
+package io.xpring.ilp;
+
+import org.interledger.spsp.server.grpc.AccountServiceGrpc;
+import org.interledger.spsp.server.grpc.BalanceServiceGrpc;
+import org.interledger.spsp.server.grpc.CreateAccountRequest;
+import org.interledger.spsp.server.grpc.CreateAccountResponse;
+import org.interledger.spsp.server.grpc.GetAccountRequest;
+import org.interledger.spsp.server.grpc.GetAccountResponse;
+import org.interledger.spsp.server.grpc.GetBalanceRequest;
+import org.interledger.spsp.server.grpc.GetBalanceResponse;
+import org.interledger.spsp.server.grpc.IlpOverHttpServiceGrpc;
+import org.interledger.spsp.server.grpc.SendPaymentRequest;
+import org.interledger.spsp.server.grpc.SendPaymentResponse;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import io.xpring.ilp.grpc.IlpJwtCallCredentials;
+import io.xpring.xrpl.XpringException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Optional;
+import java.util.Properties;
+
+/**
+ * A client that can create accounts, get accounts, get balances, and send ILP payments on a connector.
+ */
+public class DefaultIlpClient implements IlpClientDecorator {
+
+    private static String XPRING_ILP_GRPC_URL;
+    private static final Logger logger = LoggerFactory.getLogger(DefaultIlpClient.class);
+
+    // Bring in configuration properties such as GRPC url
+    static {
+        String propertiesFileName = "client.properties";
+        Properties properties = new Properties();
+
+        try {
+            // Make sure that the configuration file exists
+            URL res = DefaultIlpClient.class.getClassLoader().getResource(propertiesFileName);
+            InputStream inputStream = new FileInputStream(res.getFile());
+            // load the properties file
+            properties.load(inputStream);
+
+            XPRING_ILP_GRPC_URL = properties.getProperty("ilp.grpc.url");
+        } catch (Exception e) {
+            logger.info(String.format("Problem loading config file named %s. Using default configuration.", propertiesFileName));
+            XPRING_ILP_GRPC_URL = "http://localhost:6565";
+        }
+    };
+
+    private final BalanceServiceGrpc.BalanceServiceBlockingStub balanceServiceStub;
+    private final AccountServiceGrpc.AccountServiceBlockingStub accountServiceStub;
+    private final IlpOverHttpServiceGrpc.IlpOverHttpServiceBlockingStub ilpOverHttpServiceStub;
+
+    /**
+     * No-args Constructor.
+     */
+    public DefaultIlpClient() {
+        this(ManagedChannelBuilder
+          .forTarget(XPRING_ILP_GRPC_URL)
+          // Let's use plaintext communication because we don't have certs
+          // TODO: Use TLS!
```

Comment:
We usually put peoples names next to TODOs so there is some quick way to know who is responsible

```java
// TODO(nkramer): Use TLS!
```